android/gradle.properties

```diff
@@ -17,15 +17,14 @@
 # http://www.gradle.org/docs/current/userguide/multi_project_builds.html#sec:decoupled_projects
 # org.gradle.parallel=true
 
-android.useDeprecatedNdk=true
+# android.useDeprecatedNdk=true
 # To create a new keystore for development purposes (https://developer.android.com/studio/publish/app-signing.html)
 # $keytool -genkey -v -keystore ./status-keystore.jks -keyalg RSA -keysize 2048 -validity 10000 -alias status
 STATUS_RELEASE_STORE_FILE=~/.gradle/status-im.keystore
 STATUS_RELEASE_STORE_PASSWORD=password
 STATUS_RELEASE_KEY_ALIAS=status
 STATUS_RELEASE_KEY_PASSWORD=password
 
-# Workaround for issue https://github.com/facebook/react-native/issues/16906 (TODO: fixed in 0.57.3, remove once we upgrade react-native)
 android.enableAapt2=false
```

Comment:
Wait, shouldn't this be removed based on the comment? Or am I missing something?

Reply:
I'll try that separately. I suppose there is a lot of other garbage in this PR, because in order to make it work I had to come with many senseless suggestions and try them, and some are still here even if have no effect ¯\_(ツ)_/¯

Reply:
I see, thanks!

Reply:
app can be built without `android.enableAapt2=false`, removed